scripts/gth_config_opto.sh

```diff
@@ -31,7 +31,7 @@ do
 	then
 
 		printf "Applying special inverted RX channel config for GTH CH#$i...\n"
-		mpoke $((gth_ctrl_ch0_base_addr+256*11)) $data_ch_inverted
+		mpoke $gth_ctrl_addr $data_ch_inverted
```

Comment:
This was a change you made, or that came as a result of copying newer scripts into the repo?
(Is this one of the eponymous "scripts"?)

Reply:
from copying newer scripts.